client/jetpack-cloud/sections/partner-portal/sidebar/index.tsx

```diff
@@ -11,105 +11,100 @@ import { itemLinkMatches } from 'calypso/my-sites/sidebar/utils';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import 'calypso/components/jetpack/sidebar/style.scss';
 
-interface Props {
+type Props = {
```

Comment:
Out of curiosity, are there any benefits of using `type` here instead of `interface`?